e2etest/factory.go

```diff
@@ -48,7 +48,12 @@ type TestResourceFactory struct{}
 
 func (TestResourceFactory) GetBlobServiceURL(accountType AccountType) *blobservice.Client {
 	accountName, accountKey := GlobalInputManager{}.GetAccountAndKey(accountType)
-	resourceURL := fmt.Sprintf("https://%s.blob.core.windows.net/", accountName)
+	var resourceURL string
+	if accountName != "devstoreaccount1" {
+		resourceURL = fmt.Sprintf("https://%s.blob.core.windows.net/", accountName)
+	} else {
+		resourceURL = fmt.Sprintf("http://127.0.0.1:10000/%s/", accountName)
```

Comment:
Is this method very specific to Azurite test only? otherwise hardcoding the IP here will not work for other cases.

Reply:
only azurite has an account name of devstoreaccount1, and we don't have any tests accounts with the same name, so this is safe